### PR TITLE
fixed Hindi localization

### PR DESCRIPTION
**Pull request by aruhant**
Changes: The {{app name}} parameter of the upgrade message in Hindi had a missing set of braces around it, which was preventing it from substituting correctly. I also improved a few incorrect Hindi translations based on my proficiency as a native Hindi speaker.  

Testing: Locally tested each changed string

### CODE REVIEW

File: lib/src/upgrade_messages.dart [hunk 1]
```diff
@@ -153,7 +153,7 @@ class UpgraderMessages {
         break;
       case 'hi':
         message =
-            '{app name} का एक नया संस्करण उपलब्ध है। संस्करण {{currentAppStoreVersion}} अब उपलब्ध है-आपके पास है {{currentInstalledVersion}}.';
+            '{{app name}} का एक नया संस्करण उपलब्ध है। आपके पास संस्करण {{currentInstalledVersion}} है, लेकिन अब {{currentAppStoreVersion}} उपलब्ध है।';
```
Comment (larryaasen) [inline lib/src/upgrade_messages.dart:156]: Thanks for the PR. I like the change for the incorrect braces around app name. However, I do not like the other changes for the language text. The intent of these messages has been changed, not just simple syntax. I don't want to accept these changes unless I hear from some other Hindi speakers.

Comment (larryaasen) [reply lib/src/upgrade_messages.dart:156]: I fixed the missing braces for app name in the code so there is no need for this PR now.

Comment (larryaasen) [reply lib/src/upgrade_messages.dart:156]: @aruhant I would like to better understand the proposed changes first, which seem to change the meaning of the existing text.

Comment (aruhant) [reply lib/src/upgrade_messages.dart:156]: Alright, no problem! Those proposed changes merely use precise, more appropriate wording and do not alter the intended meaning. Though, feel free to double-check with other Hindi speakers. I am happy to clarify anything that may be unclear.

**Comment by enchntankita** [comment]
Came here looking for the braces bug. 
Thanks for merging it.
The proposed translation in the PR is much better than the previous one.

It changes the voice of the message from passive to active, and sounds much better as a native Hindi speaker.
Much clearer and easier to understand. I'd recommend merging the message.

**Comment by larryaasen** [comment]
@aruhant I'll accept these changes now, but there is a small conflict that needs to be resolved first.

**Comment by aruhant** [comment]
I have resolved those conflicts.

**Comment by larryaasen** [comment]
Thanks @aruhant - changes have been merged.

**Pull request merged**